tuf/api/metadata.py

```diff
@@ -308,14 +311,15 @@ def __init__(
 
     # Deserialization (factories).
     @classmethod
-    def from_dict(cls, signed_dict) -> 'Signed':
+    def from_dict(cls, signed_dict: JsonDict) -> 'Signed':
         """Creates Signed object from its JSON/dict representation. """
 
         # Convert 'expires' TUF metadata string to a datetime object, which is
         # what the constructor expects and what we store. The inverse operation
         # is implemented in 'to_dict'.
-        signed_dict['expires'] = iso8601.parse_date(
-                signed_dict['expires']).replace(tzinfo=None)
+        signed_dict['expires'] = datetime.strptime(
+                signed_dict['expires'],
+                "%Y-%m-%dT%H:%M:%SZ").replace(tzinfo=None)
```

Comment:
Awesome, thanks for adding this!

Reply:
We have three uses of `iso8601.parse_date` in the current codebase, so this might be a good candidate for a helper function in future.